ui/static/ui/js/manage_taxonomies.jsx

```diff
@@ -141,8 +141,7 @@ define('manage_taxonomies', ['react', 'lodash', 'jquery', 'utils', 'bootstrap'],
               <span className="utility-features">
                 <a href="#">
                   <i className="fa fa-pencil"></i>
-                </a>
-                <a data-toggle="modal" data-target="#confirm-delete"
+                </a> <a data-toggle="modal" data-target="#confirm-delete"
```

Comment:
You can confirm this by going to https://facebook.github.io/react/jsx-compiler.html and trying out:

    <div>
    <a /> <a />
    </div>

and comparing the output to

    <div>
    <a /> 
    <a />
    </div>